scripts/visualize/visualizeClimateScenarios.R

```diff
@@ -1,82 +1,95 @@
 #visualizeClimateScenarios
 library(httr)
 library(plyr)
+library(xml2)
+library(ggplot2)
 
-scenarios = c('a2', 'b1')#, 'a1b')
-
-rcp_descriptions=c("Low warming scenario (RCP 26)", "Medium warming scenario (RCP 45)", 
-                   "Medium-high warming scenario (RCP 60)", "High warming scenario (RCP 85)")
-
-read.rcp.temp = function(scenario){
+visualizeData.visualizeClimateScenarios <- function(outfile, ...){
+  
+  #outfile = 'figures/futureClimate-desktop.svg'
   
-  rcpURL = paste0('http://www.ipcc-data.org/data/ar4_tas-gm_', scenario, '.zip')
-  dl_dest = tempfile()
-  GET(rcpURL, write_disk(dl_dest, overwrite = TRUE))
-  mod_files = unzip(dl_dest, exdir=tempdir())
-  mod_names = sapply(basename(mod_files), function(x) strsplit(x, '\\.')[[1]][1])
-  all_mods = data.frame()
-  for(i in 1:length(mod_files)){
+  scenarios = c('a2', 'b1')#, 'a1b')
   
-    temps = read.table(mod_files[i], sep='')
+  rcp_descriptions=c("Low warming scenario (RCP 26)", "Medium warming scenario (RCP 45)", 
```

Comment:
Using RCP instead of A2 B1? Don't we want to show what A2 looks like since it was used in these sims?